### ISSUE TITLE
squeezezenet_ssd_lite correct rate is very low

### ISSUE THREAD

**Issue by wangnamu** [qfgaohao/pytorch-ssd#25]
hi @qfgaohao 
I trained about 100 epochs with squeezenet_ssd_lite, VOC2007 and VOC2012 for datasets, why the correct rate is very low？how to improve the correct rate？Thank you

**Comment by qfgaohao** [comment]
Hi @wangnamu @AlanStark a pretrained ImageNet model can help. Also using a smaller learning rate, as squeezenet doesn't have batchnorm layers. Increasing the number of anchors may help as well. 

Be aware, the squeeze net here is modified, as the original impl is not onnx-compatible due to the ceiling_mode in polling layers. 

If you don't have to use Squeezenet, Mobilenet v2 with width-multi 0.6 has the similar speed as Squeezenet, but has higher accuracy. I don't have the exact benchmark numbers. but it's worth to try.

**Comment by wangnamu** [comment]
hi @qfgaohao I will try your suggestion, thank you very much.
